core/src/main/java/com/datastax/oss/driver/internal/core/config/typesafe/TypesafeDriverConfig.java

```diff
@@ -130,7 +113,7 @@ public boolean reload(Config config) {
     if (rootObject.containsKey("profiles") && rootObject.get("profiles").valueType() == OBJECT) {
       ConfigObject profilesObject = (ConfigObject) rootObject.get("profiles");
       for (String profileName : profilesObject.keySet()) {
-        if (profileName.equals(DEFAULT_PROFILE_KEY)) {
+        if (profileName.equals(DriverConfigProfile.DEFAULT_NAME)) {
```

Comment:
Was thinking about this exact case "What if a user actually defined a profile named default?"  Glad this is covered here.